modules/client-configuration/pages/bootstrap-repository.adoc

```diff
@@ -2,27 +2,70 @@
 = Bootstrap Repository
 
 A bootstrap repository contains packages for installing Salt on clients, as well as the required packages for registering Salt or traditional clients during bootstrapping.
-You can create a bootstrap repository on the {productname} Server.
-
-When you have created the bootstrap repository, the packages in the repository will be installed during client registration.
-
+Bootstrap repositories are automatically created and regenerated on the {productname} Server for every synchronized product.
 
 == Prepare to Create a Bootstrap Repository
 
-Before you create the bootstrap repository, ensure client is fully synchronized with your vendor channel.
+When you select a product for synchronization, the bootstrap repo is automatically generated as soon as all mandatory channels are fully mirrored.
 
 There are two ways to check if a channel has finished synchronizing:
 
 * In the {productname} {webui}, navigate to menu:Admin[Setup Wizard] and select the [guimenu]``SUSE Products`` tab.
-This dialog displays a completion bar for each product when they are being synchronized.
+This dialog displays a completion bar for each product when they are being synchronized. Some products require also extensions to be synchronized.
 * You can also check the synchronization log file at the command prompt.
 Use the [command]``cat`` or [command]``tail -f`` command to view the [path]``/var/log/rhn/reposync/channel-label.log`` file.
 If you use this method, remember that base channels can contain multiple child channels.
 Each of the child channels will generate its own log during the synchronization progress.
 You will need to check all the base and child channel log files to be sure that the synchronization is complete.
 
+== Options for Automatic Mode
+
+=== Disable Flush Mode
```

Comment:
```suggestion
=== Flush Mode
```

Reply:
This is reference material, so it might be better to do this as a table, rather than using these third level headings.